tensorflow/contrib/lite/examples/label_image/label_image.cc

```diff
@@ -89,7 +89,7 @@ void RunInference(Settings* s) {
 
   tflite::ops::builtin::BuiltinOpResolver resolver;
 
-  tflite::InterpreterBuilder(*model, resolver)(&interpreter);
+  tflite::InterpreterBuilder (*model, resolver)(&interpreter);
```

Comment:
There should be no space here, right?

Reply:
Oops, clang-format introduced this, I guess.